Sources/Core/SoftwareBreakpointManager.cpp

```diff
@@ -65,10 +70,15 @@ ErrorCode SoftwareBreakpointManager::enableLocation(Site const &site) {
   return kSuccess;
 }
 
-ErrorCode SoftwareBreakpointManager::disableLocation(Site const &site) {
+ErrorCode SoftwareBreakpointManager::disableLocation(Site const &site,
+                                                     Target::Thread *thread) {
   ErrorCode error;
   ByteVector old = _insns[site.address];
 
+  if (thread != nullptr) {
+    DS2LOG(Warning, "thread-specific software breakpoints are unsupported");
```

Comment:
We had a conversation with this one, and I think the conclusion was that since gdb supports these, we should leave the option available.